shareprogress/services.py

```diff
@@ -0,0 +1,35 @@
+# Managing Buttons
+def create_button(page_url, button_template, page_title=None, *args):
+    # page_url: The URL of the page to be shared.
```

Comment:
One change I'd make here:

`#` will do a single-line comment in Python, whereas `"""` (3 double quotes in a row) will do a doc-block comment, which is designed for holding exactly these kinds of comments (and tends to get read by software which auto generates documents). The content of these looks great, though, I'd just condense them into a single doc-block.